vendor/vendor.json

```diff
@@ -431,6 +439,22 @@
 			"version": "v2.0.0-preview.5",
 			"versionExact": "v2.0.0-preview.5"
 		},
+		{
```

Comment:
We should find a way to keep track of our binary size to be aware of changes. Just one line in such libraries could have a big impact.

@exekias @kuisathaverat Would be great to have this as a metric in the CI job so we see in Kibana when the binary size increases (ci stats)

Reply:
Agreed, I would expect MB binary to keep growing in the short term, is that possible with jenkins already? I know we could log the binary size, but wondering if we can get it stored as a KPI we can graph.